lxd-benchmark panics when running with a remote image

When I start `lxd.benchmark` without specifying an image (defaulting to `ubuntu:`) or with a remote image (tested with `images:alpine/edge` and `images:ubuntu/22.04`), it panics:

```
$ lxd.benchmark init --count 10
Test environment:
  Server backend: lxd
  Server version: 5.5
  Kernel: Linux
  Kernel architecture: x86_64
  Kernel version: 5.15.0-46-generic
  Storage backend: dir
  Storage version: 1
  Container backend: lxc | qemu
  Container version: 5.0.1 | 7.0.0

Test variables:
  Container count: 10
  Container mode: unprivileged
  Startup mode: normal startup
  Image: ubuntu:
  Batches: 2
  Batch size: 4
  Remainder: 2

[Sep 19 15:10:29.730] Importing image into local store: e299296138c256b79dda4e61ac7454cf4ac134b43f5521f1ac894f49a9421d00
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x8c7173]

goroutine 1 [running]:
github.com/lxc/lxd/client.(*ProtocolLXD).CopyImage(_, {_, _}, {{0x0, 0xc00194d260, 0x1, {0x0, 0xedf9a2480, 0x0}, {0x0, ...}}, ...}, ...)
	/build/lxd/parts/lxd/src/client/lxd_images.go:678 +0x93
github.com/lxc/lxd/lxd-benchmark/benchmark.copyImage({_, _}, {_, _}, {{0x0, 0xc00194d260, 0x1, {0x0, 0xedf9a2480, 0x0}, ...}, ...})
	/build/lxd/parts/lxd/src/lxd-benchmark/benchmark/operation.go:74 +0x83
github.com/lxc/lxd/lxd-benchmark/benchmark.ensureImage({0x7f2fdcf15e98, 0xc000182600}, {0xa63daf, 0x7})
	/build/lxd/parts/lxd/src/lxd-benchmark/benchmark/benchmark.go:262 +0x339
github.com/lxc/lxd/lxd-benchmark/benchmark.LaunchContainers({0x7f2fdcf15e98?, 0xc000182600}, 0xa, 0x0?, {0xa63daf, 0x7}, 0x0, 0x0, 0x0)
	/build/lxd/parts/lxd/src/lxd-benchmark/benchmark/benchmark.go:50 +0xb5
main.(*cmdInit).Run(0xc00000ec00, 0x0?, {0xc00003d7e0?, 0x0?, 0x0?})
	/build/lxd/parts/lxd/src/lxd-benchmark/main_init.go:35 +0x6b
github.com/spf13/cobra.(*Command).execute(0xc0001c2780, {0xc00003d7c0, 0x2, 0x2})
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001c2500)
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
	/build/lxd/parts/lxd/src/lxd-benchmark/main.go:143 +0x707
```

It works when I download the image first and then run the benchmark with a local image.

--- a/client/lxd_images.go
+++ b/client/lxd_images.go
@@ -675,7 +675,7 @@ func (r *ProtocolLXD) CopyImage(source ImageServer, image api.Image, args *Image
 	}
 
 	// Handle profile list overrides.
-	if args.Profiles != nil {
+	if args != nil && args.Profiles != nil {
 		if !r.HasExtension("image_copy_profile") {
 			return nil, fmt.Errorf("The server is missing the required \"image_copy_profile\" API extension")
 		}

